src/quo2/components/buttons/predictive_keyboard/view.cljs

```diff
@@ -42,7 +42,8 @@
     (case type
       :words
       [rn/flat-list
-       {:data                              words
+       {:keyboard-should-persist-taps      :always
```

Comment:
This propery is required to keep the keyboard shown when a user clicks on a suggestion